src/sentry/static/sentry/app/api.tsx

```diff
@@ -129,18 +186,18 @@ export class Client {
         sudo: code === SUDO_REQUIRED,
         retryRequest: () => {
           return this.requestPromise(path, requestOptions)
-            .then((...args) => {
+            .then((data: any) => {
```

Comment:
Won't data be an object | array in general?

Reply:
Nope it won't